Makefile

```diff
@@ -0,0 +1,16 @@
+.PHONY: build bundle test bash
+
+build:
+	docker-compose build
+
+bundle:
+	docker-compose run --rm library bundle install
+
+test:
+	docker-compose run --rm library bundle exec rake
+
+bash:
+	docker-compose run --rm library bash
+
+cleanup:
+	docker-compose down
```

Comment:
I like it 👍  I saw the running containers in the past. I didn't consider them to be a problem. Still, it makes sense to add this command. Thank you!